setup.py

```diff
@@ -138,6 +138,12 @@ def _env_bool_value(env_name, default):
 # runtime, the shared library must be installed
 BUILD_WITH_SYSTEM_OPENSSL = _env_bool_value('GRPC_PYTHON_BUILD_SYSTEM_OPENSSL',
                                             'False')
+# Currently, boringssl does not support macOS arm64, so we MUST try to use the system
+# installation of openssl to build gRPC locally.
+if "darwin" in sys.platform and "arm64" == platform.machine().lower():
+    sys.stderr.write("Boringssl currently does not support macOS arm64, so we'll try to use the system "
+                     "installation of 'openssl' to build, make sure you have 'openssl' installed\n")
+    BUILD_WITH_SYSTEM_OPENSSL = True
```

Comment:
Actually, can we just change the default value here so that it is still possible to set this to `False` in the future?

Reply:
Use system installation of openssl by default on macOS arm64: https://github.com/grpc/grpc/pull/31096/commits/69aee68bd68ad3a6c79c438f2d6dfd1542253478.